api/current.ts

```diff
@@ -59,7 +58,8 @@ export default async (req: NowRequest, res: NowResponse): Promise<void> => {
       [params.lat, params.lon] = [String(lat), String(lon)];
     }
 
-    const responseData = await fetchCurrentWeather(params);
+    const {fetchCurrent} = new OpenWeather(process.env.OPENWEATHER_API_KEY || '');
```

Comment:
Not working. I wonder if non-static class methods can't be destructured 🤔

Reply:
Yep, that's the case, but I found another solution: 99e398a45f18a2959559e49eb2e234dbdfe2be05